javascript/linkedList/linkedList/linked-list.js

```diff
@@ -4,92 +4,76 @@ const Node = require("./node");
 
 class LinkedList {
   constructor(value) {
-    this.value = value;
     this.head = null;
   }
 
   isEmpty() {
-    return this.head === null ? true : false;
+    return this.head === null;
   }
 
+  // Add a Node to the front of a linked list
   insert(value) {
-    //insert a node at the beginning of a given LL
-
-    if (this.isEmpty() === true) {
-      // the list is empty
-      // make a new Node with the value given
-      const node = new Node(value);
-      //   // assign this.head to that new node
-      this.head = node;
-    } else if (this.isEmpty() === false) {
-      // the list is NOT empty
-      // make a new Node with the value given
-      const node = new Node(value);
-      // make the new Node's .next point at the current head
+    const node = new Node(value);
+    if (this.isEmpty()) {
       node.next = this.head;
-      // reassign the head to the new node
-      this.head = node;
     }
+
+    this.head = node;
   }
-  append(value) {
-    // add a value to the end of a linked list
-    const node = new Node(value);
-    // traverse the entire list until I find the one whose next is null
 
+  // Add a Node to the end of a linked list
```

Comment:
I moved the function's description above the function. It is very uncommon to find a description below.